x-pack/test_utils/jest/config.js

```diff
@@ -9,6 +9,7 @@ import { RESERVED_DIR_JEST_INTEGRATION_TESTS } from '../../../src/dev/constants'
 export default {
   rootDir: '../../',
   roots: [
+    '<rootDir>/plugins',
```

Comment:
is this file used? `x-pack/scripts/jest` uses another config file https://github.com/elastic/kibana/blob/fa3b8c0207e4d5d5f4d828f883dad5c7a77938f4/x-pack/dev-tools/jest/create_jest_config.js#L9

Reply:
It is, but only by the integration test config. The `jest` command in xpack does not, and use the file you linked instead 🙈 

https://github.com/elastic/kibana/blob/bf04235dae35452061cc7ea3d86d96c19a58206c/x-pack/test_utils/jest/config.integration.js#L8-L11